packages/checklist-service/test/services/checklists/add-collaborator.test.js

```diff
@@ -1,31 +1,38 @@
-const proxyquire = require('proxyquire')
-const { test } = require('tap')
+const t = require('tap')
+const { v4: uuid } = require('uuid')
 
-const { userId } = require('../../fixtures')
-
-const received = {}
-
-const addCollaboratorHandler = proxyquire(
-  '../../../services/checklists/add-collaborator.js',
-  {
-    './checklist': {
-      addCollaborator: params => {
-        received.addCollaboratorParams = params
-        return received
-      }
+let addCollaboratorParams = {}
+let addCollaboratorReturnVal = {}
+const addCollaboratorHandler = t.mock('../../../services/checklists/add-collaborator.js', {
```

Comment:
use `t.mock` as it achieves the same functionality as `proxyquire` but without an additional dependency